woocommerce/woocommerce-ios
# Fix message in the console about missing xib in WPAuthenticator

Closes #3542 

⚠️  The PR depends on a PR in WordPressAuthenticator: wordpress-mobile/WordPressAuthenticator-iOS#571 so it is pointing the reference to that library in the podfile to a branch. The reference will be pinned to a version after the PR in WPAuthenticator is merged ⚠️ 

## Changes
* Point WordPressAuthenticator to the branch fixing the warnings.

## How to test
* Checkout develop first. 
* Logout
* Start the login flow. Select Login With Store Address
* Enter a wp.com address
* Notice the following errors logged in the console when reaching the email screen:

```
[Storyboard] Unknown class _TtC31WordPressAuthenticatorResources19SpacerTableViewCell in Interface Builder file.
Error: Unidentified tableViewCell type found.
```

* Now checkout this branch
* Run `bundle exec pod install`
* Log out
* Start the login flow, select Login With Store Address
* Enter a wp.com store address
* Check the console when reaching the email screen. The two lines mentioned above should not be in the console anymore
* Enter an email address that does not match the store address. Navigate back to the email screen.
* Check the two lines above are not in the Console log


Update release notes:
